docs/self-managed/components/modeler/web-modeler/configuration/logging.md

```diff
@@ -26,6 +26,24 @@ By default, Web Modeler's `restapi` component logs in JSON. For a more readable
 SPRING_PROFILES_INCLUDE=default-logging
 ```
 
+### Changing log level at runtime
+
+You can adjust log levels dynamically using Spring Boot Actuator [loggers](https://docs.spring.io/spring-boot/api/rest/actuator/loggers.html) endpoints:
+
+```bash
+curl 'http://localhost:8091/actuator/loggers/io.camunda' \
+  -i -X POST \
+  -H 'Content-Type: application/json' \
+  -d '{"configuredLevel":"DEBUG"}'
+```
+
+Replace `io.camunda` with the logger you want to adjust.
+
+For
```

Comment:
```suggestion

```

Reply:
Thanks. Initially I was going to add a paragraph to explain more about the Spring loggers endpoint, but then just updated the initial phrase to include a link to it.